README.md

```diff
@@ -494,8 +494,10 @@ scrollbar.scrollIntoView(elem, options?): void
 
 | Property | type | default | description |
 | :-------: | :--: | :-----: | :---------- |
-| `offsetTop` | number | `0` | Offset to top edge of container. |
-| `offsetLeft` | number | `0` | Offset to left edge of container. |
+| `alignToTop` | boolean | `true` | Whether to align to the top or the bottom edge of container. |
+| `offsetTop` | number | `0` | Offset to top edge of container (used only if alignToTop is true). |
+| `offsetLeft` | number | `0` | Offset to left edge of container (used only if alignToTop is true). |
```

Comment:
Oh you are right, it actually does take effect, it's just a mistake in the documentation. Will fix soon.